internal/provider/resource_user_test.go

```diff
@@ -0,0 +1,61 @@
+package provider
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+)
+
+func TestAccResourceUser(t *testing.T) {
+	const (
+		name     = "test user"
+		role     = "member"
+		newRole  = "administrator"
+		userType = "service"
+	)
+	resource.Test(t, resource.TestCase{
+		PreCheck:                 func() { testAccPreCheck(t) },
+		ProtoV6ProviderFactories: testAccProtoV6ProviderFactories,
+		Steps: []resource.TestStep{
+			// Create and Read testing
+			{
+				Config: testAccResourceUser(name, role, userType),
+				Check: resource.ComposeAggregateTestCheckFunc(
+					resource.TestCheckResourceAttr("humanitec_user.test", "name", name),
+					resource.TestCheckResourceAttr("humanitec_user.test", "role", role),
+					resource.TestCheckResourceAttr("humanitec_user.test", "type", userType),
+				),
+			},
+			// ImportState testing
+			// {
+			// 	ResourceName: "humanitec_value.app_val1",
+			// 	ImportState:  true,
+			// 	ImportStateIdFunc: func(s *terraform.State) (string, error) {
+			// 		return fmt.Sprintf("%s/%s", appID, key), nil
+			// 	},
+			// 	ImportStateVerify: true,
+			// },
```

Comment:
As importing is supported, could you uncomment this case?

Reply:
Fixed